crates/lib/src/bootloader.rs

```diff
@@ -87,7 +92,39 @@ pub(crate) fn install_systemd_boot(
     Command::new("bootctl")
         .args(["install", "--esp-path", esp_path.as_str()])
         .log_debug()
-        .run_inherited_with_cmd_context()
+        .run_inherited_with_cmd_context()?;
+
+    match autoenroll {
+        None => return Ok(()),
+        Some(SecurebootKeys { dir, keys }) => {
```

Comment:
Slightly more idiomatic as 
```
if let Some(SecurebootKeys { dir, keys}) = autoenroll {
  ...
}
```

Then in the None case we just naturally fall through.

One reason this is more idiomatic is that if someone wanted to add more code after this, that'd work just fine.